src/http/controllers/device.js

```diff
@@ -1,12 +1,20 @@
 import deviceService from '../../services/device';
 import deviceValidation from '../validations/device';
 
+/**
+ * @description Controller for "update admin user" API operation
```

Comment:
Copy cater 😄 . This ain't `update admin user` controller.

Reply:
fixed